tensorflow_addons/seq2seq/loss_test.py

```diff
@@ -297,6 +297,17 @@ def testZeroWeights(self):
             compare_total = np.zeros((self.batch_size, self.sequence_length))
             self.assertAllClose(compare_total, res)
 
+    def testAmbiguousOrder(self):
+        with self.assertRaises(ValueError):
```

Comment:
```suggestion
        with self.assertRaisesRegexp(ValueError, 'because of ambiguous order'):
```

Reply:
@sjmielke Would you mind modifying the test case in this way and then we can merge :)